Tools/gdb/libpython.py

```diff
@@ -709,13 +709,18 @@ def write_repr(self, out, visited):
             pyop_value.write_repr(out, visited)
         out.write('}')
 
+    try:
+        _gdb_error = gdb.error
+    except AttributeError:
+        _gdb_error = RuntimeError
```

Comment:
Good idea.  BTW, can you point me where gdb.error is defined?  I cloned binutils-gdb, but could not find the right place.  Nevertheless I can confirm the inheritance:

```
(gdb) python print gdb.error.mro()
[<class 'gdb.error'>, <type 'exceptions.RuntimeError'>, <type 'exceptions.StandardError'>, <type 'exceptions.Exception'>, <type 'exceptions.BaseException'>, <type 'object'>]
```